### ISSUE TITLE
Overview page: Chart colors

### ISSUE THREAD

**Issue by nlcwong** [project-koku/koku-ui#241]
The trend line colors are hard to see on a black background.

1. Go to Cost Management > Overview

![trendlinecolors](https://user-images.githubusercontent.com/31672086/47588159-4d67ae00-d933-11e8-9902-28f622fb0708.png)

Would like to adopt PF4 color palatte for charts and apply them to the customized charts in Koku for the interim until we get the PF4 chart components available to use in Koku.

**Comment by christiemolloy** [comment]
@dlabrecq would you be able to change the chart colors? Just checked with Michael Celedonia about the new colors.

![pf4 chart colors](https://user-images.githubusercontent.com/20118816/48150684-69514500-e28d-11e8-8a2a-1bf37c8e895e.png)

**Comment by christiemolloy** [comment]
PF4 Chart Colors


Purple

100 - #CBC1FF
200 - #B2A3FF
300 - #A18FFF
400 - #8476D1
500 - #6753AC


Blue

100 - #7DC3E8
200 - #39A5DC
300 - #007BBA
400 - #00659C
500 - #004D76


Cyan

100 - #8BB4B9
200 - #5C979D
300 - #2E7981
400 - #005C66
500 - #00434B


Green

100 - #A2DA9C
200 - #88D080
300 - #6EC664
400 - #519149
500 - #3C6C37


Yellow

100 - #FBE8B6
200 - #FADD92
300 - #F9D67A
400 - #DCAB56
500 - #AC7F33


Orange

100 - #F4B678
200 - #EF9234
300 - #EC7A08
400 - #C46100
500 - #8F4700

**Comment by dlabrecq** [comment]
I'd really like to see these colors added to pf-next so we don't hard code hex values. These values really should have global variables assigned to them. Ideally, the variables would also be available via @patternfly/react-tokens, in order for react components (i.e., charts) to make use of them. If there is any change, it won't break our code.

That said, I've implemented a @patternfly/react-charts package, which has just been merged. If the color variables are added to pf-next, we can swap out the charts for the components in @patternfly/react-charts.
